src/lib/input/input-container.ts

```diff
@@ -302,6 +336,12 @@ export class MdInputContainer implements AfterContentInit {
     this._mdInputChild._placeholderChange.subscribe(() => this._validatePlaceholders());
   }
 
+  ngAfterViewInit() {
+    // Avoid animations on load.
+    this._messageAnimationState = 'enter';
+    this._changeDetectorRef.detectChanges();
```

Comment:
The point is that the animation shouldn't run on the first load, which is why it gets added after the view init. It also needs to trigger change detection, in order to avoid "<thing> was changed after it was checked" errors.